### DISCUSSION TITLE
Updated french translation for chrome extention v9.0.23

### DISCUSSION THREAD

**Discussion by ted1958** [aclap-dev/video-downloadhelper#911] [Language & Translation]
I've just added two missing strings in the french translation. I send you the txt file. Thanks.
[videodownloadhelper_extension_chrome_tradfr.txt](https://github.com/user-attachments/files/15930809/videodownloadhelper_extension_chrome_tradfr.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
